BuildScript.Nuke/GenerateSbom/SolutionSbomGenerator.cs

```diff
@@ -116,25 +118,56 @@ public void Generate (Tool cycloneDxTool, IReadOnlyCollection<ProjectInfo> proje
 
       _outputFile.Parent.CreateDirectory();
 
-      File.Copy(_workingDirectory / cleanedSbomFileName, _outputFile, overwrite: true);
+      //The sbom cleaner only works with xml files, so we need to convert the sbom if we want a different format.
+      var outputSbomFileToCopy = _workingDirectory / cleanedSbomFileName;
+      if (_outputFile.Extension.EndsWith("json"))
```

Comment:
`.EndsWith` feels a bit strange here. Why do we want to support extensions like `blablajson`? I would suggest using `_outputFile.Extension == ".json"` instead